### PR TITLE
chart.js Labels support string[] and string[][]

### PR DESCRIPTION
**Pull request by zigomir**
As seen here http://www.chartjs.org/samples/latest/scales/multiline-labels.html. [source, see line 29](view-source:http://www.chartjs.org/samples/latest/scales/multiline-labels.html).

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: `view-source:http://www.chartjs.org/samples/latest/scales/multiline-labels.html`
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ declare namespace Chart {
     }
 
     interface ChartData {
-        labels?: string[];
+        labels?: string[] | string[][];
```
Comment (rbuckton) [inline types/chart.js/index.d.ts]: I think you want `(string | string[])[]`, given the linked example.

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts (hunk 1 above)
Comment (zigomir) [inline types/chart.js/index.d.ts]: You're right, thanks. Will change.

Comment (zigomir) [inline types/chart.js/index.d.ts]: @rbuckton done.

File: types/chart.js/index.d.ts [hunk 2]
```diff
@@ -130,7 +130,7 @@ declare namespace Chart {
     }
 
     interface ChartData {
-        labels?: string[] | string[][];
+        labels?: (string | string[])[];
```
Comment (rbuckton) [inline types/chart.js/index.d.ts]: My apologies, it looks like our tslint rules don't like complex array types using `[]`. You need to change the definition again to `labels?: Array<string | string[]>;` for the build to pass.

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts (hunk 2 above)
Comment (zigomir) [inline types/chart.js/index.d.ts]: No problem. I saw linting fail, but had some difficulties running it locally 😊 .

**Review by rbuckton — APPROVED** [review]

**Comment by rbuckton** [comment]
Thanks for the contribution!

**Pull request merged**